packages/twenty-front/src/pages/object-record/RecordIndexPage.tsx

```diff
@@ -20,27 +20,19 @@ const StyledIndexContainer = styled.div`
 export const RecordIndexPage = () => {
   const objectNamePlural = useParams().objectNamePlural ?? '';
 
-  const { objectNameSingular } = useObjectNameSingularFromPlural({
-    objectNamePlural,
-  });
-
-  const { createOneRecord: createOneObject } = useCreateOneRecord({
-    objectNameSingular,
-  });
-
   const recordIndexId = objectNamePlural ?? '';
   const setHotkeyScope = useSetHotkeyScope();
 
   const { setSelectedTableCellEditMode } = useSelectedTableCellEditMode({
     scopeId: recordIndexId,
   });
 
+  const { setPendingRecordId } = usePendingRecordId();
+
   const handleAddButtonClick = async () => {
-    await createOneObject?.({
-      position: 'first',
```

Comment:
not sure what that is for, did not use it afterwards

Reply:
it's to add the record to the top in the case of an un-ordered list (list are always ordered by position after all other sorts), we need it!